04/app.js

```diff
@@ -2,8 +2,22 @@ const user = {
     firstName: 'Adam',
     lastName: 'Nowak',
     born: {
-        day: '14',
+        day: '18',
         month: '04',
         year: '1985'
     }
+}
+
+var dateObj = new Date();
+var month = dateObj.getUTCMonth() + 1; //months from 1-12
+var day = dateObj.getUTCDate();
+
+console.log(month, day)
+console.log(parseInt(user.born.month), parseInt(user.born.day))
+
+if (parseInt(user.born.day) === day && parseInt(user.born.month) === month) {
+    console.log('User ma dzisiaj urodziny!')
+}
+if (parseInt(user.born.day) !== day || parseInt(user.born.month) !== month) {
```

Comment:
Powielasz ten sam błąd. Jeśli masz już `if` to do przeciwnego warunku używasz `else`.
Robienie tego samego warunku dwa razy mija się z celem i jest nie optymalne

Reply:
ok, tutaj else w ogóle nie jest potrzebne, wystarczy jeden warunek (jeśli są urodziny)